src/commands/start.ts

```diff
@@ -41,11 +46,44 @@ export async function handler(
 
   const app = await createServer(config.port, config);
   debug('Start server on port %d', config.port);
-  return new Promise(resolve => {
-    app.listen(config.port, async () => {
+  return new Promise((resolve, reject) => {
+    const serverStartedSuccessfully = async () => {
       printRouteInfo(config);
       resolve();
-    });
+    };
+    const handleServerError = async (error: ServerError) => {
+      if (error.code === 'EADDRINUSE') {
+        const answers = await inquirer.prompt([
+          {
+            type: 'input',
+            default: config.port + 1,
```

Comment:
Do we need to valid that +1 will keep the port within 0..65535?

Reply:
Also good point! I've made this a random port number now.